test/archipelago-controller.spec.ts

```diff
@@ -66,6 +70,28 @@ describe("archipelago controller", () => {
     await expectIslandsInControllerWith(controller, ["1", "2"], ["3"])
   })
 
+  it("should forward positions and receive updates", async () => {
```

Comment:
Is the name of this test on purpose? Seems like it was copy pasted

Reply:
Fixed